Jenkinsfile

```diff
@@ -247,13 +247,11 @@ pipeline {
             model.train_ds.batch_size=8 \
             model.validation_ds.batch_size=8 \
             trainer.max_epochs=1 \
-            +trainer.max_steps=1 \
```

Comment:
Why did we remove max_steps=1?

Reply:
basically it calculates validation accuracy on the full validation dataset, this is SQuAD specific

Reply:
max_steps is only for training steps.
num_sanity_val_steps is for validation steps, which in lightning 0.9 is now -1 and does a full run.
I'm pretty sure you can add max_steps=1 back in

Reply:
@yzhang123 Could you open a PR after this to see if we can add back max_steps=1?

Reply:
@blisc oh yes, the ci had also num_sanity_val_steps limited, so probably thats why it broke